libs/components/core/src/lib/modules/scrollable-host/scrollable-host.service.ts

```diff
@@ -206,34 +210,56 @@ export class SkyScrollableHostService {
 
   public watchScrollableHostClipPathChanges(
     elementRef: ElementRef,
+    additionalContainers: Observable<ElementRef[]> = of([]),
```

Comment:
The clip path was not working when AG Grid used auto-height layout b/c the grid body element was not scrollable in that case.